helm/cluster-azure/values.yaml

```diff
@@ -76,11 +76,15 @@ internal:
     softEvictionGracePeriod: memory.available=30s,nodefs.available=2m,nodefs.inodesFree=1m,imagefs.available=2m,pid.available=1m
     hardEvictionThresholds: memory.available<200Mi,nodefs.available<10%,nodefs.inodesFree<3%,imagefs.available<10%,pid.available<20%
     evictionMinimumReclaim: imagefs.available=5%,memory.available=100Mi,nodefs.available=5%
+  image:
+    gallery: gsCAPITest1-5cb24dcf-a2d0-4aba-820f-b52ca78f96e6
+    name: '{{ printf "capi-flatcar-stable-%s-gen2" .Values.internal.kubernetesVersion }}'
+    version: latest
   identity:
     # Only required until we have AZWI for CAPZ and only required on MC cluster
     attachCapzControllerIdentity: false
 
     # TODO: This is temporary so we can deploy clusters without dedicated UA Identity until the operator for WC clusters are ready
     # https://github.com/giantswarm/giantswarm/issues/25039
     enablePerClusterIdentity: false
-  kubernetesVersion: 1.24.8
```

Comment:
Please note that values.yaml is generated off values.schema.json now. So the default must be modified in the schema, too.

Reply:
yep, i need to do a big rebase of this PR :)